tests/pylorax/test_gitrpm.py

```diff
@@ -111,9 +111,10 @@ def git_fail_repo_test(self):
             ref="v1.1.0"
             destination="/srv/testing-rpm/"
         """ % ("/tmp/no-repo-here/"))
-        with self.assertRaises(RuntimeError):
+        with self.assertRaises(RuntimeError) as cm:
```

Comment:
Small nitpick, sorry, we use 'as e' for error results, not 'cm'. Looks good other than that.

Reply:
Sure, `e` works.